src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -363,6 +363,18 @@
   <data name="PX1080Title" xml:space="preserve">
     <value>Data view delegates should not start long-running operations</value>
   </data>
+  <data name="PX1055Fix1" xml:space="preserve">
```

Comment:
@saloponov please use more context in names. Digits don't tell anything.The fix is better to be named after the action it does. For example the first fix name could be `PX1055RemoveAllKeysExceptIdentityFix` and so on